src/app/containers/Article/index.jsx

```diff
@@ -34,11 +34,14 @@ class ArticleContainer extends Component {
   }
 
   render() {
-    const { data } = this.props;
+    const { data, match } = this.props;
     const { content, metadata, promo } = data;
+    const { params } = match;
+    const { id } = params;
```

Comment:
Rather than extract `id` out of the `match` object, we should extract the ID out of of `metadata.id` in the [data feed](https://github.com/bbc/simorgh/blob/latest/data/scenario-01.json#L3). This will be used elsewhere in metadata.

Reply:
@jtart Currently, since we are using `scenario-xx` values, this method will fail. We'd need to go through the data feed and change the `c0000000001o` to `scenario-01`, etc.